lms/config/__init__.py

```diff
@@ -22,6 +22,8 @@ def configure(settings):
         "google_client_id": sg.get("GOOGLE_CLIENT_ID"),
         "google_developer_key": sg.get("GOOGLE_DEVELOPER_KEY"),
         "google_app_id": sg.get("GOOGLE_APP_ID"),
+        "onedrive_client_id": sg.get("ONEDRIVE_CLIENT_ID"),
+        "onedrive_secret": sg.get("ONEDRIVE_CLIENT_SECRET"),
```

Comment:
So this is a single client ID and secret that we'll use for all customers? We don't need different client IDs and secrets for different users?

Reply:
Yeah it will be one key per "onedrive application" so we could use one for all schools the "Hypothesis One Drive" app or create one app per school if that made sense? I don't see an immediate benefit of multiple apps.